migrations/v2.js

```diff
@@ -0,0 +1,182 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+/**
+ * ? Should we be adding changes to course aria attributes? - https://github.com/adaptlearning/adapt-contrib-accordion/commit/6dca48baf740d7117ab987378ce4270c8580395b
+ */
+describe('adapt-contrib-accordion - v1.0.0 > v2.0.0', async () => {
+  let accordions;
+
+  whereFromPlugin('adapt-contrib-accordion - from v1', { name: 'adapt-contrib-accordion', version: '<=2.0.0' });
```

Comment:
@joe-replin should this be `<2.0.0`?

Reply:
Interesting question. I'm migrating to v2.0.0 so I guess I'm migrating from everything from v1.0.0 and up. I'm guessing I shouldn't be attempting to migrate from below v2.0.0 and my migrations should start at the point in which I am migrating v2.0.0 > v2.0.4? Is that the consensus?

Reply:
The describe block should not run if already at v2.0.0 hence < instead of <=

Reply:
I agree. My question is whether we should include describe blocks for changes introduced in v2.0.0, or if we should start our migrations from the first version after v2.0.0. In this case, that would be v2.0.4.

Reply:
Cool, I don't know. @joe-allen-89 any thoughts?

Reply:
The pattern I've followed in all my scripts so far is as follows.

If there is some `v2.a.b` to `v2.c.d` that has an attribute change then that will go in `v2.js`. So I'd say yes if there is some attribute change from v2.0.0 to v2.x.x then include it in `v2.js`.

On the other hand if an attribute change occurs between the last release of v2 and, say, v3 then I would put the migration in v3.js because the change happens when you move to v3.

Reply:
Not that I can see many people still on v1 versions but it's probably best we don't try and migrate anything lower than the first script. 

I think the best course of action is to create a range rather than just use < for the first script to prevent migrating older versions, the two options would be:

`whereFromPlugin('adapt-contrib-accordion - from v1', { name: 'adapt-contrib-accordion', version: '>=1.1.4 <2.0.0' });`
or remove 2.0.0 and change the following whereFromPlugin to
`whereFromPlugin('adapt-contrib-accordion - from v1', { name: 'adapt-contrib-accordion', version: '>=2.0.0 <2.1.0' });`

As 1.1.4 doesn't have a FW version listed I'd be inclined to go for option 2.

Reply:
So just to be clear, we shouldn't be supportting v1 FWs. If I went with your option 2, I would remove everything from this from from line 4 - line 108. And then we would start where the migration goes from v2.0.0 > v2.0.4.

Reply:
I've removed the describe for migrations from v1 to v2.